README.md

```diff
@@ -73,8 +77,8 @@ else
 end
 ```
 
-You can run this code from [examples/basic.rb](examples/basic.rb).
+You can find this code in [the examples directory](https://github.com/momentohq/client-sdk-ruby/tree/main/examples).
 
 ## Contributing
 
-See our [instructions for CONTRIBUTING](./CONTRIBUTING.md).
+See our [instructions for CONTRIBUTING](https://github.com/momentohq/client-sdk-ruby/blob/main/CONTRIBUTING.md).
```

Comment:
just curious why absolute here?

Reply:
When YARD generates the HTML it does not know how to turn it into an absolute URL.

Reply:
so YARD is going to generate stuff from the README as well as the ruby code?  If that's the case, TIL, 🚢

Reply:
Yes. For example, YARD's own README.

https://github.com/lsegal/yard/blob/main/README.md
https://rubydoc.info/gems/yard/file/README.md